book/src/04_core_vitss/api/v0.yaml

```diff
@@ -477,6 +477,17 @@ paths:
 components:
   schemas:
     Fund:
+      description: 'Note, ANY date-time formatted field in this API may report:
+
+        "1970-01-01T00:00:00Z" 
+        
+        Which is The unix Epoch.        
+        This date is to be interpreted that the date-time for the event the 
+        field represents is not yet known.
+
+        The UI may either not display anything for this field, or TBD or an 
+        equivalent message.'
+
```

Comment:
probably not for this PR, but this seems like something that should be represented by `null` (or some equivalent construct)...

Reply:
It should,  this is for backward compatibility with vit-ss which can only represent dates in the DB for this field.  I need to rationalize the API which will be another step.